### ISSUE TITLE
Detect compression/packing and throw a warning

### ISSUE THREAD

**Issue by mirh** [LongSoft/IFRExtractor-RS#19]
And only that, it's not like the tool would have to handle it. 
It's really just a quality of life feature, to hint the noobest users that their file might be returning null/empty/unknown results because they extracted the PE wrong.

**Comment by NikolajSchlej** [comment]
I do not think that catering for the noobest users makes sense for this kind of tools. If somebody really wants to, a wrapper script can do all that just fine.

**Comment by mirh** [comment]
IFRExtractor is almost a necessary stop for any wannabe bios tinkerer (and yes, bios modding certainly isn't for the lightest of heart and "smart", but most of times we are just talking about finding the variable store offsets to flip one bit).
...and honestly it happens even to the best of us to press *extract body* (instead of *extract body uncompressed*) in UEFItool and then taking minutes to figure out why nothing is being detected.

This said, I just checked a few compressed sections here and there and now I'm not sure this is even doable? Like, to the very least I couldn't find one that had the same starting header/number of another.
